realm/realm-jni/build.gradle

```diff
@@ -81,6 +89,34 @@ if (project.clang instanceof String) {
 if (project.forceDownloadCore instanceof String) {
     ext.forceDownloadCore = Boolean.parseBoolean(project.forceDownloadCore)
 }
+if (project.stripSymbols instanceof String) {
```

Comment:
You are right. I even found this tweet :) https://twitter.com/kohsukekawa/status/575671438809636865